kingfisher_scrapy/spiders/colombia.py

```diff
@@ -29,14 +29,21 @@ class Colombia(LinksSpider):
         If ``from_date`` is provided and ``until_date`` don't, defaults to today.
     """
     name = 'colombia'
-    next_page_formatter = staticmethod(parameters('page'))
+    next_page_formatter = staticmethod(parameters('_id'))
     default_from_date = '2011-01-01'
 
+    @classmethod
+    def from_crawler(cls, crawler, *args, **kwargs):
+        spider = super(Colombia, cls).from_crawler(crawler, date_format='date', *args, **kwargs)
+        if (spider.from_date or spider.until_date) and hasattr(spider, 'year'):
+            raise scrapy.exceptions.CloseSpider('The use of from and/or until with year parameter is not supported')
```

Comment:
Make the names of spider arguments more explicit:

```suggestion
            raise scrapy.exceptions.CloseSpider('You cannot specify both a year spider argument and from_date/until_date spider argument(s).')
```